src/components/CardList/Item.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import Card from '../Card';
+import { INodeItem } from './CardList';
+
+export interface Props {
```

Comment:
interface properties on lines 5 - 14 should be separated by a semi-colon instead of a comma.
Ex:
`id: number;` not `id: number,`